target/linux/ath79/dts/ar9344_openmesh_om5p.dts

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "openmesh,om5p", "qca,ar9344";
+	model = "OpenMesh OM5P";
+
+	chosen {
+		/delete-property/ bootargs;
+	};
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_power_blue;
+		led-failsafe = &led_power_blue;
+		led-running = &led_power_blue;
+		led-upgrade = &led_power_blue;
+		label-mac-device = &eth1;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_lan_wan_blue_pin>;
+
+		led_power_blue: power_blue {
+			label = "blue:power";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		wifi_red {
+			label = "red:wifi";
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi_yellow {
+			label = "yellow:wifi";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi_green {
+			label = "green:wifi";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		lan_blue {
+			label = "blue:lan";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wan_blue {
+			label = "blue:wan";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
```

Comment:
LAN and WAN GPIO are swapped here compared to ar71xx? Intentional?

Reply:
It was mislabeled in ar71xx. If you do following, you will observe a blinking **WAN** LED (which is wrong in my opinion)

    t=0; while true; do t=$((!t)); echo $t > /sys/class/leds/om5p\:blue\:lan/brightness ; sleep 1; done

If you do following on ath79, you will see the **LAN** LED blinking:

    t=0; while true; do t=$((!t)); echo $t > /sys/class/leds/blue\:lan/brightness ; sleep 1; done